Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
+        },
+        copy: {
+            // Copy external front end dependencies into appropriate directories
+            frontEndDependencies: {
+                files: [
+                    // D3
+                    {expand: true, cwd: "./node_modules/d3/", src: "**", dest: ".//src/lib/ext/d3/"},
```

Comment:
Placing them in `/lib/ext` makes it easier to filter them for `.gitignore` and other purposes; however, it would be not much more difficult to do it on a per library basis (or in this case, ignore `src/lib` except for Infusion).

The double slashes are a typo. :)